product_lot_sequence/models/stock_move.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+import logging
```

Comment:
Why is the logger stuff added in a migration commit?
Is the logger needed at all?

Reply:
@bosd, I remove logging import